src/components/atoms/DateTimePicker/DateTimePicker.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import { Story } from '@storybook/react'
-import dayjs from 'dayjs'
+import moment from 'moment'
```

Comment:
why we switch to use `moment`? 🤔

Reply:
@quy-truong Customize component using this function

```
const AntDatePicker = generatePicker<Dayjs>(dayjsGenerateConfig)
```

somehow doesn't pick up the global prefix configuration nor the local. We make it work first then improve later